src/translations/en.json

```diff
@@ -2,5 +2,16 @@
     "policy_notification.NotificationLanguage.none": "Language of notification",
     "policy_notification.NotificationLanguage.label": "Language of notification",
     "policy_notification.languageOfNotification.null": "Language of notification",
-    "policy_notification.notificationApproval": "Approve notification"
+    "policy_notification.notificationApproval": "Approve notification",
```

Comment:
There is a project in Lokalize (https://app.lokalise.com/project/983881975d551c5fdba845.54008358) with keys from to define language packs. Can you add the translations there? Thanks.

Reply:
Translations put in Lokalise